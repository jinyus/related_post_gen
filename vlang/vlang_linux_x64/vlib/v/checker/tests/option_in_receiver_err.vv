struct Foo {
mut:
	name string
}

fn (mut f Foo) t0(arr ?[]int) ?string {
	return arr?.len.str()
}

fn (mut f ?Foo) t1(arr ?[]int) ?string {
	return arr?.len.str()
}