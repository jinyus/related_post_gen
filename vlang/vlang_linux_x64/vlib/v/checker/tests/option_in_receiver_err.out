vlib/v/checker/tests/option_in_receiver_err.vv:10:9: error: option types cannot have methods
    8 | }
    9 | 
   10 | fn (mut f ?Foo) t1(arr ?[]int) ?string {
      |         ~~~
   11 |     return arr?.len.str()
   12 | }
